pkg/detectors/codacy/codacy_test.go

```diff
@@ -1,6 +1,3 @@
-//go:build detectors
```

Comment:
Hey Roxanne, could you put these tags back?

You can run these tests by adding `-tags=detectors` with your `go test` command.

Reply:
Could you also update these docs with that? https://github.com/trufflesecurity/trufflehog/tree/main/hack/docs